test/requests/base_request_test.rb

```diff
@@ -42,4 +42,15 @@ def test_locate
 
     assert_instance_of(SyntaxTree::CallNode, parent)
   end
+
+  # We can remove this once we drop support for Ruby 2.7
```

Comment:
@vinistock I moved this into `base_request` since it's not really specific to `SemanticHighlighting`.